config/chunk-uploader.php

```diff
@@ -118,4 +118,49 @@
 
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Resumable.js Options
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure the options for the Resumable.js driver.
+    |
+    */
+
+    'resumable-js' => [
+
+        // The name of the multipart request parameter to use for the file chunk
+        'param' => 'file',
+
+        //  HTTP method for chunk test request.
+        'test-method' => \Illuminate\Http\Request::METHOD_GET,
+        //  HTTP method to use when sending chunks to the server (POST, PUT, PATCH).
+        'upload-method' => \Illuminate\Http\Request::METHOD_POST,
+
+        // Extra prefix added before the name of each parameter included in the multipart POST or in the test GET.
+        'parameter-namespace' => '',
+
+        'parameter-names' => [
```

Comment:
Very minor suggestion that I feel.

```suggestion
        'parameters' => [
```
```suggestion
        'options' => [
```

Reply:
I wouldn't use `options` because that is not explicit enough. I was also thinking between `parameters` and `parameter-names` then I decided to use the later because `parameters` could contain `namespace` config as well but that in my opinion should be separated for convenience.